pkg/metrics/nextgengrafana/tidb_summary.jsonnet

```diff
@@ -0,0 +1,167 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+local grafana = import 'grafonnet/grafana.libsonnet';
+local dashboard = grafana.dashboard;
+local row = grafana.row;
+local graphPanel = grafana.graphPanel;
+local prometheus = grafana.prometheus;
+local template = grafana.template;
+
+local myNameFlag = 'DS_TEST-CLUSTER';
+local myDS = '${' + myNameFlag + '}';
+
+// A new dashboard
+local newDash = dashboard.new(
+  title='Test-Cluster-TiDB-Summary',
+  editable=true,
+  graphTooltip='shared_crosshair',
+  refresh='30s',
+  time_from='now-1h',
+)
+.addInput(
```

Comment:
Could we add a comment like: `Add data source selector`

Reply:
Done